doc/radosgw/config-ref.rst

```diff
@@ -416,6 +382,73 @@ instances or all radosgw-admin commands can be put into the ``[global]`` or the
 :Default: ``true``
 
 
+Garbage Collection Settings
+===========================
+
+The Ceph Object Gateway allocates storage for new and overwritten objects
+immediately. Additionally, the parts of a multi-part upload also consume
+storage.
+
+The Ceph Object Gateway purges the storage space used for deleted objects in the
+Ceph Storage cluster some time after the gateway deletes the objects from the
+bucket index. Similarly, the gateway will delete data associated with a
+multi-part upload after the multi-part upload completes or when the upload has
+gone inactive or failed to complete for a configurable time frame. The process
```

Comment:
This sentence about multipart can be dropped, it is not automatic, only if configured explicitly via bucket lifecycle etc will the multipart upload be dropped when it is inactive, so it is better not mentioning this.